data/otf-data.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  loss: {
+    sum: ['area__ha'],
+    groupBy: ['umd_tree_cover_loss__year'],
```

Comment:
As discussed shall we try to find a way to abstract these to a constants file and reference them in data/analysis-cached, data/forest-types and data/land-categories?

Reply:
added a new file "temporary" where we can define these layers, will update the migration document of cached analysis where we will combine these next week! AJ will take a look and make sure we use the correct layers then I will patch all widgets that need this